README.md

```diff
@@ -159,12 +176,14 @@ Or if your app is configured to detach the validator by default, you can force t
 ```
 npm run dev -- -attach-validator
 ```
+- `-a` can also be used.
 
-Run your app on two CPUs (default is `max`):
+Configure how many CPUs your app will run with. Can set to `max` to use all available CPUs. (Default is 1)
```

Comment:
"Configure how many CPUs your app will run on (default is `max`)."

yes we want the default to be max